pkg/store/promproxy.go

```diff
@@ -75,6 +78,18 @@ func (p *PrometheusProxy) Info(ctx context.Context, r *storepb.InfoRequest) (*st
 	return res, nil
 }
 
+func (p *PrometheusProxy) getBuffer() []byte {
+	b := p.buffers.Get()
+	if b == nil {
+		return make([]byte, 0, 32*1024)
```

Comment:
can we have constant explaining why exactly this amount

Reply:
I dunno lol :)